azure-pipelines.yml

```diff
@@ -18,6 +18,7 @@ resources:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```

Comment:
I am confused^^ Did you merge my test branch? If so, please revert this line :-)

Reply:
Yes that's what I did. Also merged in latest master to resolve a merge conflict. And now removed this line